packages/integral_isolates/lib/src/integral_isolates.dart

```diff
@@ -58,6 +60,24 @@ class Isolated extends StatefulIsolate {
             backpressureStrategy ?? NoBackPressureStrategy() {
     if (autoInit) init();
   }
+
+  /// Creates a minimal isolate that requires a specific input type [R].
+  ///
+  /// If [backpressureStrategy] is set, this instance will use provided
+  /// strategy. If is is not provided [NoBackPressureStrategy] will be used as
+  /// default.
+  ///
+  /// If [autoInit] is set to false, you have to call function [init] before
+  /// starting to use the isolate.
+  static _TailoredStatefulIsolate<Q, R> tailored<Q, R>({
+    BackpressureStrategy<Q, R>? backpressureStrategy,
+    bool autoInit = true,
+  }) {
```

Comment:
Hmm, uncertain if this is a good way to go...
I would either use a factory here instead (but I guess that isn't possible since `_TailoredStatefulIsolate` doesn't `implement/extend` `StatefulIsolate`), or make the `_TailoredStatefulIsolate` public and make the user use the constructor there instead.

Reply:
Thanks for the valuable feedback! I have now decided to go with two distinct classes instead and totally deprecated the Isolated class :) It turned out much cleaner in the end.